src/ot_span.erl

```diff
@@ -32,9 +32,12 @@
 -include("opentelemetry.hrl").
 
 -type start_opts() :: #{parent => undefined | opentelemetry:span() | opentelemetry:span_ctx(),
-                        sampler => ot_sampler:sampler(),
+                        attributes => opentelemetry:attributes(),
+                        sampler := ot_sampler:sampler(),
```

Comment:
this being mandatory means there are no defaults I guess?

Reply:
Na, that was wrong. Accidentally taken from the SDK repo, where it is required to be set at the time of calling the function in `ot_span_utils`. Fixing.